cmd/flags_network.go

```diff
@@ -58,6 +58,12 @@ var (
 		Value: metadata.DefaultNetwork.AccessPolicyOracleAddress,
 	})
 
+	mmnAddressFlag = altsrc.NewStringFlag(cli.StringFlag{
+		Name:  "mmn-address",
```

Comment:
This looks a bit cryptic to me, and also it contains a dash. How about `mymysterium.url`?